src/app/cluster/details/shared/version-change-dialog/component.ts

```diff
@@ -58,6 +62,14 @@ export class VersionChangeDialogComponent implements OnInit, OnDestroy {
       .pipe(takeUntil(this._unsubscribe))
       .subscribe(project => (this.project = project));
     this._googleAnalyticsService.emitEvent('clusterOverview', 'clusterVersionChangeDialogOpened');
+
+    this._machineDeploymentService
+      .list(this.cluster.id, this.project.id)
+      .pipe(map(nodes => nodes.filter(node => node.spec.dynamicConfig)))
+      .pipe(takeUntil(this._unsubscribe))
+      .subscribe(nodes => {
+        this.nodesSupportDynamicKubeletConfig = nodes.map(node => ` MD-${node.name}`).toString();
```

Comment:
Instead of `toString` you can use `.join(', ')` which adds extra space after `,` for better readability.